### PR TITLE
Vapor 4 release

### LINKED ISSUE #72: Vapor 4 update
Potentially starting a new project in the not too distant future that will need to use Shopify. Hoping to use Vapor and would be great to be able to use this package with Vapor 4.

### PR DESCRIPTION
**Pull request by 0xTim**
Vapor 4 release. 

Resolves #72

### CODE REVIEW

**Comment by 0xTim** [comment]
@calebkleveter I'm aware that there are a couple of missing services, but those may have to come in future releases as I need to get a release out. A review would be awesome! I know GitHub and Google work

File: Sources/ImperialCore/Helpers/String+Tools.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import RoutingKit
+
+extension String {    
+	var pathComponents: [PathComponent] {
+		var pathComponentArray = [PathComponent]()
+		if let components = URL(string: self)?.pathComponents {
+			for item in components where item != "/" {
+				pathComponentArray.append(PathComponent(stringLiteral: item))
+			}
+		}
```
Comment (calebkleveter) [inline Sources/ImperialCore/Helpers/String+Tools.swift:11]: I would probably write this:

```swift
return URL(string: self)?.pathComponents.compactMap { component -> PathComponent? in
    guard component != "/" else { return nil }
    return PathComponent(stringLiteral: component)
} ?? []
```
But if you want to leave it, go ahead. It's just a style difference.

**Review by calebkleveter — COMMENTED** [review]
Looking good! I just left a couple of minor nits. Do we want to remove Facebook support while we're at it, just because of the security and privacy issues that they entail?

**Comment by 0xTim** [comment]
@calebkleveter sorry I missed this before the tag! In terms of FB, I think it's a popular one to request so we should probably keep it in (and we can't remove it now it's been released anyway!)

**Pull request merged**